qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -174,13 +179,26 @@ def write_local_config(self):
             export TEST_DIR={}
             export SCRATCH_DEV={}
             export SCRATCH_MNT={}
-            export CEPHFS_MOUNT_OPTIONS="-o name=admin,secret={}"
+            export CEPHFS_MOUNT_OPTIONS="-o {}name=admin,secret={}"
```

Comment:
Would it be better if options are kept after `name` and `secret`?

Reply:
That's okay. I can fix it.